README.md

```diff
@@ -50,6 +50,8 @@ A lot of the functionality of the `teal` framework derives from the following pa
 
 See these packages for more information about how to use the different parts of the `teal` framework.
 
+Please see [`teal` gallery](https://github.com/insightsengineering/teal.gallery) and [TLG Catalog](https://github.com/insightsengineering/tlg-catalog) to see examples of `teal` apps.
```

Comment:
Is https://insightsengineering.github.io/teal.gallery/main/ better?

Reply:
hmmm you might be right but currently all our references are to the GH (not GitHubPages) so I wanted to stay consistent to the currently existing approach